src/Screen/Fields/Quill.php

```diff
@@ -41,8 +41,9 @@ class Quill extends Field
      * @var array
      */
     protected $attributes = [
-        'value'  => null,
-        'height' => '300px',
+        'value'   => null,
+        'toolbar' => null,
```

Comment:
What do you think to list by default all available groups here?

```php
'toolbar' => ["text", "color", "header", "list", "format", "media"]
```

Reply:
Yes, this is a good idea, I did not think about it.